all/build.gradle

```diff
@@ -1,11 +1,5 @@
 description = "OpenCensus All"
 
-buildscript {
```

Comment:
Why remove this?

Reply:
Question is why was there? This is not needed because we don't load any plugin here, maybe I am wrong but I think this is useless here.